src/theme/components/featuredContent.ts

```diff
@@ -46,37 +46,57 @@ const FeaturedContent = defineMultiStyleConfig({
             bgColor: "dark.ui.bg.default",
           },
           img: screenreaderOnly(),
+          containerType: "inline-size",
+          "@container (min-width: 0px)": {
+            " > div": {
+              flexDirection: imageAtEnd ? "column-reverse" : "column",
+            },
+            "[data-image]": {
+              height: "320px",
+              width: "100%",
+            },
+          },
+          "@container (min-width: 568px)": {
```

Comment:
Can you explain a bit why this one is needed and then the 600px one?

Also, I think you and Olivia already mentioned this, but you cannot interpolate a variable into the strings, right? It'd be nice to use the values from the breakpoint object and set this as 
```
[`@container (min-width: ${breakpoints.md})`]
```

Reply:
There is 1rem padding around the components on Storybook pages, so when the image is full width the featured content container is 600px since there is no padding. I think we can use 600px as a standard.

It is possible to interpolate the variable, I will update this.